doc/using_procfiles.rst

```diff
@@ -19,7 +19,7 @@ source tree that contains zero or more lines of the form::
     <process type>: <command>
 
 The ``process type`` is a string which may contain alphanumerics and underscores
```

Comment:
Would you be able to update the prose here as well as the regex?

```
The ``process type`` is a string which may contain alphanumerics, as well as
underscores and dashes (``[A-Za-z0-9_-]+``), and uniquely... [etc.]
```

Reply:
Thanks for spotting that!